templates/taxi/car_detail.html

```diff
@@ -2,21 +2,42 @@
 
 {% block content %}
   <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
+    Car model: {{ car.model }}
+  </h1>
+  
+  <p><strong>Manufacturer:</strong> {{ car.manufacturer.name }} ({{ car.manufacturer.country }})</p>
+  
+  <div class="ml-3">
+    <h4>
+      Drivers
+    
+      {% if user not in car.drivers.all %}
```

Comment:
The condition `{% if user not in car.drivers.all %}` may not work as expected because `user` is a Django `User` object, and `car.drivers.all` returns a queryset of `Driver` objects. Consider using `{% if user.pk not in car.drivers.values_list('pk', flat=True) %}` to compare IDs instead.